style.css

```diff
@@ -626,7 +634,7 @@ body {
   .card-work-1 {
     margin: 142px;
     flex-direction: row;
-    height: 496px;
+    height: 50%;
     left: 0;
     top: 0;
     border-radius: 24px;
```

Comment:
- [ ] The popup should follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins)

Your desktop design   | Figma desktop design
------------- | -------------
 <img width="921" alt="Screenshot 2022-10-13 at 21 40 50" src="https://user-images.githubusercontent.com/77038610/195679621-e7707e7a-1980-49c9-8107-d18de2152a4d.png"> |  <img width="507" alt="Screenshot 2022-10-13 at 21 41 10" src="https://user-images.githubusercontent.com/77038610/195679688-ec130201-ab45-4e5b-abd7-a7d59b38dd7b.png">

Your mobile design  | Figma mobile design
------------- | -------------
<img width="288" alt="Screenshot 2022-10-13 at 21 42 46" src="https://user-images.githubusercontent.com/77038610/195680004-a805bea8-7615-4145-bdee-4b3de6b63357.png"> | <img width="221" alt="Screenshot 2022-10-13 at 21 43 32" src="https://user-images.githubusercontent.com/77038610/195680141-bb03345c-6846-4f7c-8dfc-a57837ee2527.png">

Please make sure that you have the same layout, eg., text placement margins paddings etc.